airflow/www_rbac/views.py

```diff
@@ -707,21 +707,34 @@ def __del__(self):
         region_name = os.environ["AWS_REGION"]
 
         logs = [ s3_file["Key"] for s3_file in paginated_list(s3_prefix, bucket, region_name) ]
-        log_names = [ item.split("/")[-1].replace(".log", "") for item in logs ]
-        log_paths_str = '"' + '","'.join(logs) + '"'
-        log_names_str = '"' + '","'.join(log_names) + '"'
+        all_log_s3_names = [ item.split("/")[-1].replace(".log", "") for item in logs ]
+        # log_paths_str = '"' + '","'.join(logs) + '"'
+        log_s3_names_str = '"' + '","'.join(all_log_s3_names) + '"'
+
+        group_1_log_s3_names = []
+        group_2_log_s3_names = []
+
+        for log_name in all_log_s3_names:
+            if len(log_name) <= 3:
```

Comment:
Will update this in next PR - there are better ways to separate and group logs (based on filename or filter by prefix/suffix in s3)